packages/serial/serial.0.1.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+synopsis: "Serial communication module"
+maintainer: ["m-laniakea@github.com"]
+authors: ["eir"]
+license: "GPL-2.0-only"
+homepage: "https://github.com/m-laniakea/oserial"
+bug-reports: "https://github.com/m-laniakea/oserial/issues"
+depends: [
+  "dune" {>= "2.5"}
+  "ocaml" {>= "4.08"}
+  "lwt" {>= "4.0.0"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```

Comment:
```suggestion
  ["dune" "subst"] {dev}
```
This is a bug in dune < 2.5, there is nothing to do on your side, it is enough to fix it here.